Package.swift

```diff
@@ -1014,6 +1020,12 @@ if ProcessInfo.processInfo.environment["SWIFTPM_LLBUILD_FWK"] == nil {
     ]
 }
 
+if ProcessInfo.processInfo.environment["SWIFTPM_DOCC_ENABLED"] != nil {
+    package.dependencies += [
```

Comment:
suggestion: I don't think we'll need to guard this dependency on swift-package-manager, might be better to just remove this :)

Reply:
(meant to mark this as "Requested Changes"!)

Reply:
Using this comment to mark this as a request for the above changes.

Reply:
This dependency makes all the build slower even if it's not used